collections/Douce/MS_Douce_311.xml

```diff
@@ -30,61 +30,559 @@
+                           <item>fol. 2v: Retable</item>
+                           <item>fol. 21v: Pilgrims’ badges</item>
+                           <item>fol. 29v: Jewels</item>
+                           <item>fol. 30r: Scenes of a boating party and the gathering of manna</item>
+                           <item>fol. 35r: Men carrying grapes</item>
+                           <item>fol. 68r: Woman with two infants, sitting astride a lion</item>
+                           <item>fol. 74r: Pilgrims’ bottles</item>
+                           <item>[lacking between fols. 80 and 81: Peacock feathers, now <ref target="https://collections.vam.ac.uk/item/O1012781/leaf-from-a-book-of-manuscript-cutting/">London, Victoria and Albert Museum, Prints, Drawings, and Paintings Collection, no. 4101</ref>]</item>
+                           <item>fol. 98r: Church portal</item>
+                           <item>fol. 101r: Architecture</item>
+                           <item>fol. 123r: Rosary beads</item>
+                        </list>
+                     </decoNote>
+                     <decoNote type="rubrication">Rubrication.</decoNote>
                   </decoDesc>
                   <bindingDesc>
-                     <binding notBefore="1800" notAfter="1900">
-                        <p>Green morocco (early 19th cent. ?) with fine gold ornament on gilt clasps:
-         on one side in the centre is 'T.W.' with 'Deus alit me', on the other the arms of Williams
-         [i.e. Theodore Williams vicar of Hendon].</p>
+                     <binding notAfter="1826" notBefore="1813">
+                        <p>Bound by <persName key="person_17145938" role="bnd">Charles Lewis (1786–1836)</persName> in green morocco with fine gold ornament on gilt clasps, in imitation of the French Duodo style: see Giles Barber and David Rogers, <q>Bindings from Oxford Libraries: II. A <q>Duodo</q> Pastiche Binding by Charles Lewis</q>, <title>Bodleian Library Record</title>, 8.3 (1969): 140–43. On one side in the centre is <quote>T.W.</quote> with <quote>Deus alit me</quote>, on the other the arms of Theodore Williams. The Williams sale catalogue of 1827 describes this as <quote>a matchless specimen of the art, by Charles Lewis; it is of green morocco, tooled entirely over after a pattern of a book belonging to Diana of Poitiers</quote> (<ref target="https://hdl.handle.net/2027/nyp.33433057513875?urlappend=%3Bseq=120">p. 104</ref>). The book is accompanied by a box case, also covered in morocco.</p>
                      </binding>
                   </bindingDesc>
                </physDesc>
                <history>
                   <origin>
-                     <origDate calendar="Gregorian" notAfter="1500" notBefore="1488">after 1488</origDate>
+                     <origDate calendar="Gregorian" notAfter="1499" notBefore="1490">1490s</origDate>
```

Comment:
I think it would be worth saying more about the grounds for dating (e.g. "As-Vijvers suggests a date ... on stylistic and iconographical grounds, comparing ...")